example/otel-collector/README.md

```diff
@@ -134,98 +50,28 @@ need to create the Jaeger and Prometheus exporters:
 
 ```yml
 ...
-    exporters:
-      jaeger:
-        endpoint: "jaeger-collector.observability.svc.cluster.local:14250"
-
-      prometheus:
-           endpoint: 0.0.0.0:8889
-           namespace: "testapp"
+exporters:
+  jaeger:
+    endpoint: "jaeger:14250"
+    tls:
+      insecure: true
+  prometheus:
+    endpoint: 0.0.0.0:8889
 ...
 ```
 
-## OpenTelemetry Collector service
-
-One more aspect in the OpenTelemetry Collector [configuration](./k8s/otel-collector.yaml) worth looking at is the NodePort service used for accessing it:
-
-```yaml
-apiVersion: v1
-kind: Service
-metadata:
-        ...
-spec:
-  ports:
-  - name: otlp # Default endpoint for otlp receiver.
-    port: 4317
-    protocol: TCP
-    targetPort: 4317
-    nodePort: 30080
-  - name: metrics # Endpoint for metrics from our app.
-    port: 8889
-    protocol: TCP
-    targetPort: 8889
-  selector:
-    component: otel-collector
-  type:
-    NodePort
-```
-
-This service will bind the `4317` port used to access the otlp receiver to port `30080` on your cluster's node. By doing so, it makes it possible for us to access the Collector by using the static address `<node-ip>:30080`. In case you are running a local cluster, this will be `localhost:30080`. Note that you can also change this to a LoadBalancer or have an ingress extension for accessing the service.
-
-# Running the code
-
-You can find the complete code for this example in the [main.go](./main.go)
-file. To run it, ensure you have a somewhat recent version of Go (preferably >=
-1.13) and do
-
-```bash
-go run main.go
-```
-
-The example simulates an application, hard at work, computing for ten seconds
-then finishing.
-
 # Viewing instrumentation data
 
 Now the exciting part! Let's check out the telemetry data generated by our
 sample application
 
 ## Jaeger UI
 
-First, we need to enable an ingress provider. If you've been using microk8s,
-do
-
-```bash
-microk8s enable ingress
-```
-
-Then find out where the Jaeger console is living:
-
-```bash
-kubectl get ingress --all-namespaces
-```
-
-For us, we get the output
-
-```
-NAMESPACE       NAME           CLASS    HOSTS   ADDRESS     PORTS   AGE
-observability   jaeger-query   <none>   *       127.0.0.1   80      5h40m
-```
-
-indicating that the Jaeger UI is available at
-[http://localhost:80](http://localhost:80). Navigate there in your favorite
+Jaeger UI is available at
+[http://localhost:16686](http://localhost:16686). Navigate there in your favorite
```

Comment:
For Jaeger and Prometheus, how about giving a couple indications on the generated data, and what to look for rather than only the link to the interfaces?

Jaeger could mention to look into `test-service` go view the generated trace. Prometheus could give a sample query.

Reply:
@dmathieu I've updated as your suggestion 👍